### ISSUE TITLE
 I made the same changes to your training code and test code, but it failed during the test. 

### ISSUE THREAD

**Issue by longmalongma** [hkchengrex/STCN#38]
Very strange, I made the same changes to your training code and test code, but it failed during the test. Do you know why? The error is as follows: But I can successfully test by adding False to prop_model.load_state_dict(prop_saved), but such a test will cause the test accuracy to drop.

![image](https://user-images.githubusercontent.com/40291722/128293786-10afd097-afe8-4175-984f-f166ac187fbe.png)

![image](https://user-images.githubusercontent.com/40291722/128293853-5fb15f8b-ddd3-4536-924b-443f9861b3e2.png)

**Comment by hkchengrex** [comment]
If you changed the network structure, you would also need to change this file: https://github.com/hkchengrex/STCN/blob/main/model/eval_network.py

**Comment by longmalongma** [comment]
> If you changed the network structure, you would also need to change this file: https://github.com/hkchengrex/STCN/blob/main/model/eval_network.py

Yes, I made the same changes to this file but the problem still exists.

**Comment by hkchengrex** [comment]
Well then maybe those changes are problematic. That's all I can tell from the error messages.
